app/components/elements/Userpic.jsx

```diff
@@ -20,7 +20,7 @@ class Userpic extends Component {
         } catch (e) {}
 
         if (url && /^(https?:)\/\//.test(url)) {
-            const size = width && width > 48 ? '320x320' : '72x72'
+            const size = width && width > 48 ? '320x320' : '120x120';
```

Comment:
Can we try 96x96? Since 48px is the cutoff, with retina @2x the optimal size would be 96px